File: prak205/PRAK205-2010817320004-SalsabilaSyifa.c
#include <stdio.h>
#include <math.h>

int main(){

    float a,b,keliling,luas;
	printf("Masukkan Nilai tinggi segitiga : ");
	scanf("%f",&a);
	printf("Masukkan Nilai sisi miring segitiga : ");
	scanf("%f",&b);

    float hypotenusa;
     hypotenusa=sqrt(b*b-a*a);
    keliling = a + b + hypotenusa;
    luas = a * hypotenusa / 2;

    printf("Alas = %.0f \n",hypotenusa);
    printf("Tinggi = %.0f \n",a);
    printf("Keliling = %.0f \n",keliling);
    printf("Luas = %.0f \n",luas);

     return 0;
}

File: prak205/PRAK205-2010817320004-SalsabilaSyifa.py
import math
from math import sqrt

A = float(input("Masukan nilai tinggi segitiga : "))
B = float(input("Masukan nilai sisi miring segitiga : "))

C = sqrt(B*B - A*A)

print("Alas =", format(C,'.0f'))
print("Tinggi =", format(A,'.0f'))

keliling = A + B + C
print("Keliling =", format(keliling,'.0f'))

luas = (C * A)/2
print("Luas =", format(luas,'.0f'))